draft-ietf-taps-interface.md

```diff
@@ -1848,23 +1848,22 @@ while also simultaneously trying to establish a Connection from the Local
 Endpoint candidates to the Remote Endpoint candidates.
 
 If there are multiple Local Endpoints or Remote Endpoints configured, then
-initiating a `Rendezvous` action will systematically probe the reachability
+initiating a `Rendezvous` action will cause the Transport Service
+Implementation to systematically probe the reachability
```

Comment:
```suggestion
implementation to systematically probe the reachability
```

Reply:
Actually, we more commonly call it the "Transport Services system" instead of "implementation... but this is nitpicking, I believe.

Reply:
We split System into Implementation + API, no?

Reply:
Ah, I see, Implementation is capitalized in -arch but not in -impl and mixed in api... I'll fix this with a separate PR then.